test/extended/oauth/token.go

```diff
@@ -27,42 +27,6 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer] OAuth Authenticator", func()
 	oc := exutil.NewCLI("oauth-access-token-e2e-test")
 	ctx := context.Background()
 
-	g.It(fmt.Sprintf("accepts classic non-prefixed access tokens"), func() {
```

Comment:
does this imply an inverse test when we switch over?

A blocker BZ would be good not to forget.

Reply:
we won't be able to e2e test that since we won't be able to create such a token, but might still be worth a unit test